### ISSUE TITLE
Selected file support adjust border-top-width to make it more outstanding

### ISSUE THREAD

**Issue by jeffreyyjp** [microsoft/vscode#190471]
![image](https://github.com/microsoft/vscode/assets/17985592/f59b2e0c-23ab-4a20-ab87-4a7ff0f22abe)

I think the current one is too thin.

**Comment by jeffreyyjp** [comment]
![image](https://github.com/microsoft/vscode/assets/17985592/fd421c9d-efb8-4bb8-b49a-a81bedaeee38)

Why my another project doesn't have border-top-width with color?

**Comment by daviddossett** [comment]
Tried bumping to 2px to see what this might look like. I'll run it by the team for feedback:

<img width="1380" alt="CleanShot 2023-08-21 at 09 19 04@2x" src="https://github.com/microsoft/vscode/assets/25163139/0419479e-4a47-498e-ab29-4221cfc0da63">



> Why my another project doesn't have border-top-width with color?

I'm not actually sure. Do you have any color customizations in your `settings.json` that might change the border color?

**Comment by jeffreyyjp** [comment]
@daviddossett I don't do any color customizations about border color.

**Comment by daviddossett** [comment]
Any other notable differences between projects? Safe to assume it's the same computer/OS/VS Code version?

**Comment by jeffreyyjp** [comment]
@daviddossett Yes, same computer/OS/VS Code version, just different project. I don't even know where to configure border color or border width.

**Comment by daviddossett** [comment]
Sounds like a bug then. Some other people on the VS Code team reported seeing this sporadically too. Opened #191101

**Comment by bpasero** [comment]
As for the bug (if any), I need assistance to reproduce in https://github.com/microsoft/vscode/issues/191101

**Comment by benibenj** [comment]
We fixed the bug a couple of months ago
